projects/components/src/table/data/table-cdk-data-source.ts

```diff
@@ -54,6 +54,7 @@ export class TableCdkDataSource implements DataSource<TableRow> {
     this.buildChangeObservable()
       .pipe(
         tap(() => this.loadingStateSubject.next({ loading$: NEVER })),
+        debounceTime(100),
```

Comment:
can we add a comment about the reasons for the debounce so that we don't forget later? :)

Reply:
Please refer to #326